library_db_data.json

```diff
@@ -0,0 +1,193 @@
+[
+  {
+    "model": "books.author",
+    "pk": 1,
+    "fields": {
+      "first_name": "J.K.",
+      "last_name": "Rowling",
+      "pseudonim": null
+    }
+  },
+  {
+    "model": "books.author",
+    "pk": 2,
+    "fields": {
+      "first_name": "George",
+      "last_name": "Orwell",
+      "pseudonim": null
+    }
+  },
+
+  {
+    "model": "books.author",
+    "pk": 1,
+    "fields": {
+      "first_name": "Ernest",
+      "last_name": "Hemingway",
+      "pseudonim": null
+    }
+  },
+  {
+    "model": "books.book",
+    "pk": 1,
+    "fields": {
+      "title": "The Old Man and the Sea",
+      "author": [1],
+      "cover": "HARD",
+      "image": "book_cover/old_man_sea.jpg",
+      "inventory": 8,
+      "daily_fee": 3
+    }
+  },
+  {
+    "model": "books.author",
+    "pk": 3,
+    "fields": {
+      "first_name": "Haruki",
+      "last_name": "Murakami",
+      "pseudonim": null
+    }
+  },
+  {
+    "model": "books.book",
+    "pk": 2,
+    "fields": {
+      "title": "Kafka on the Shore",
+      "author": [3],
+      "cover": "SOFT",
+      "image": "book_cover/kafka_on_the_shore.jpg",
+      "inventory": 12,
+      "daily_fee": 2.5

```

Comment:
We define daily_fee like IntegerField

Reply:
@Zenionoi it must be decimal

Reply:
Its true, but we have `daily_fee = models.IntegerField()` and nowhere work to transform dec to int

Reply:
so, we should change it)))